docs/ru/innopolis_open_L22_AERO.md

```diff
@@ -0,0 +1,183 @@
+# Innopolis Open 2020
+
+
+## Команда 
+<ul style="list-style: none; font-size:15px; padding-left:10px">
```

Comment:
HTML is not allowed in the Clover documentation (except few cases). Please, use markdown syntax instead:

```
* item 1
* item 2
```